### DISCUSSION TITLE
Interactively debug Windows build

### DISCUSSION THREAD

**Discussion by reikdas** [community/community#26432] [Actions]
<p>Is there a way to interactively debug a Windows build?<br>
I know that it is possible for Linux and macOS using <a href="https://github.com/marketplace/actions/debugging-with-tmate" rel="noopener nofollow ugc">https://github.com/marketplace/actions/debugging-with-tmate</a></p>
<p>I am aware that there has been some discussion on this like here - <a href="https://github.community/t/debugging-windows-build-failures/17169" class="inline-onebox">Debugging Windows build failures</a> but since that was some time ago, I was wondering if there is a way to do it now?</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/reikdas">@reikdas</a>,</p>
<p>The following two actions for interactive debugging can support Windows runners:</p>
<ul>
<li><a href="https://github.com/marketplace/actions/debug-via-ssh" rel="noopener nofollow ugc"><strong>Debug via SSH</strong></a></li>
<li><a href="https://github.com/marketplace/actions/safe-action-debugger" rel="noopener nofollow ugc"><strong>Safe Action Debugger</strong></a></li>
</ul>

**Comment by Simran-B** [comment]
<p>There’s also this:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/nelsonjchen/reverse-rdp-windows-github-actions" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/431;"><img src="https://user-images.githubusercontent.com/7819991/181097635-c29386e2-bf2c-4728-8074-ec49d479ba10.png" class="thumbnail" width="690" height="431" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/d/3df6e64dc26cb0f44905bf6a36d4026ab8357680_2_690x431.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/d/3df6e64dc26cb0f44905bf6a36d4026ab8357680_2_1035x646.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/3/d/3df6e64dc26cb0f44905bf6a36d4026ab8357680_2_1380x862.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/d/3df6e64dc26cb0f44905bf6a36d4026ab8357680_2_10x10.png"></div>

<h3><a href="https://github.com/nelsonjchen/reverse-rdp-windows-github-actions" target="_blank" rel="noopener nofollow ugc">nelsonjchen/reverse-rdp-windows-github-actions</a></h3>

<p>Reverse Remote Desktop into Windows on GitHub Actions - nelsonjchen/reverse-rdp-windows-github-actions</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
